Makefile.PL

```diff
@@ -360,6 +360,9 @@ if (-d 'inc/patches') {
         for my $file (glob 'inc/patches/*.patch') {
             # replacing \n with \n will change \n to \r\n on Win, without this patch.exe will crash
             system "cd \Q$mxbase\E && perl -pe \"s/\\n/\\n/\" ../../\Q$file\E | patch -p1 -N";
+
+            # some Windows systems don't like the path escape ...
+            # system "cd $mxbase && perl -pe \"s/\\n/\\n/\" ../../$file | patch -p1 -N";
```

Comment:
Is this issue still happens on your system? Which Windows and Perl versions do you have on this system? Maybe you're running this using some other shell instead of standard command.com (like cygwin or powershell)?